ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/ConditionalFlows.java

```diff
@@ -0,0 +1,31 @@
+package io.syndesis.qe.pages.integrations.editor.add.steps;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ConditionalFlows {
+
+    private static final class Elements {
+        //temporary xpath, change to css datatestid if present - should be in the next build
```

Comment:
I think this comment can be deleted, the selector now uses `testid`

Reply:
fixed, thanks